CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ else()
     set(ENABLE_REQUIRES_ALL_SYMBOLS OFF)
 endif()
 
-option(ENABLE_COVERAGE "Enable compiling tests with coverage." ON)
+option(ENABLE_COVERAGE "Enable compiling tests with coverage." OFF)
```

Comment:
For my info, why toggle this?

Oh, I see - this now changes the production build as well? So locally, we should make sure to toggle this ON when developing, right?

Reply:
I believe this should be OFF while developing in order to get the benefit of the assertions.